### PR TITLE
Incoming cases

### PR DESCRIPTION
**Pull request by LindsayYoung**
[View incoming cases as 'New'](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/25)

## What does this change?
Adds a status that defaults to new
## Screenshots (for front-end PR):
Screen Shot 2019-11-14 at 5 39 48 PM

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

Did a desk check with Aviva and Jacklynn.

Merge https://github.com/usdoj-crt/crt-portal/pulls first. I didn't add the generated css to this PR so we have one less merge conflict to deal with.

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/static/sass/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -258,6 +261,40 @@ form#report-form {
...
+}
+
+// crt view statuses
+.status-new {
+  color: $white !important;
+  background-color: $blue-warm-vivid-70;
+  padding: 4px;
+  padding-left: 10px;
+  padding-right: 10px;
+  border-radius: 8.5rem;
```
Comment (alexsoble) [inline crt_portal/static/sass/_uswds-theme-custom-styles.scss]: Hmmm ... `8.5rem` == `136px`, which is too high for a `border-radius`. 

I would re-check the intended `border-radius` with Aviva, it looks fairly small and subtle in the mockup.

File: crt_portal/static/sass/_uswds-theme-custom-styles.scss [hunk 2]
```diff
@@ -258,6 +261,40 @@ form#report-form {
...
+}
+
+.status-open {
+  background-color: $gold-20v;
+  font-weight: bold;
+  padding: 4px;
+  padding-left: 10px;
```
Comment (alexsoble) [inline crt_portal/static/sass/_uswds-theme-custom-styles.scss]: Our spacing values are generally in increments of `8px` (`0.5rem`), but maybe this is an exception in the annotated mockup?

**Comment by LindsayYoung** [comment]
We made changes based on design pairing yesterday. This seems like a good thing to circle back on for confirmation at standup with Aviva and team. 

Noted it in the card: https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/25

**Comment by LindsayYoung** [comment]
@alexsoble just wanted to check to see if you had any remaining questions on this PR. We should get this merged before we start a new PR that has model changes.

**Pull request merged**